server/methods.js

```diff
@@ -1,15 +1,52 @@
 Meteor.startup(function() {
   Meteor.methods({
     addComment: function(initiativeId, input) {
+      check(initiativeId, String);
```

Comment:
According to the docs, the second parameter to check(...) is:

    check(key, allowedCount) checks a key, if less than allowedCount of the (unexpired) records exist, it passes

So with that in mind, what's the significance of passing *String* as the parameter?